test/supervisor.js

```diff
@@ -145,6 +145,21 @@ describe('supervisor', function(done) {
     });
   });
 
+  describe('log decoration', function() {
+    var TAGS_WORKER = /^pid:\d+ worker:\d+ .+/;
```

Comment:
This should probably be `worker:1` since `\d` will match the supervisor and if `worker:1` doesn't match then there's something else wrong.